content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -27,15 +27,34 @@ Meanwhile, when IP based NetworkPolicies are created, we define policies based o
 
 Network policies are implemented by the [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/). To use network policies, you must be using a networking solution which supports NetworkPolicy. Creating a NetworkPolicy resource without a controller that implements it will have no effect.
 
-## Isolated and Non-isolated Pods
+## What Communication is Allowed
```

Comment:
There is a lot of existing NetworkPolicy discussion/documentation that uses the concepts of isolated and non-isolated pods, and I think at this point it is best to stick with the existing concepts.